timeforcoffee.xcodeproj/project.pbxproj

```diff
@@ -754,7 +757,7 @@
 					};
 					EC46C5271BB5C78A00F2A963 = {
 						CreatedOnToolsVersion = 7.0;
-						DevelopmentTeam = DN76797UYF;
+						DevelopmentTeam = 6EMM22S2YK;
 						SystemCapabilities = {
```

Comment:
can you please move the DevTeam back them and the PROVISIONING_PROFILE changes below as well